Docs/Configuration.tex

```diff
@@ -6728,26 +6717,25 @@ \subsection{Quirks Properties}\label{uefiquirkprops}
   amount of cores present on the platform, the recommended starting value is \texttt{500000}.
 
   This is an experimental quirk, which should only be used for the aforementioned problem.
-  In all other cases the quirk may render the operating system unstable and is not recommended.
+  In all other cases, the quirk may render the operating system unstable and is not recommended.
   The recommended solution in the other cases is to install a kernel driver such as
   \href{https://github.com/RehabMan/VoodooTSCSync}{VoodooTSCSync},
   \href{https://github.com/interferenc/TSCAdjustReset}{TSCAdjustReset},
   or \href{https://github.com/lvs1974/CpuTscSync}{CpuTscSync} (a more specialised
   variant of VoodooTSCSync for newer laptops).
 
-  \emph{Note}: The reason this quirk cannot replace the kernel driver is
-  because it cannot operate in ACPI S3 mode (sleep wake) and because the UEFI firmware
-  provides very limited multicore support preventing the precise update of the MSR
-  registers.
+  \emph{Note}: This quirk cannot replace the kernel driver because it cannot operate in
+  ACPI S3 (sleep wake) mode and because the UEFI firmware only provides very limited
+  multicore support which prevents precise updates of the MSR registers.
 
 \item
   \texttt{UnblockFsConnect}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
   \textbf{Failsafe}: \texttt{false}\\
   \textbf{Description}: Some types of firmware block partition handles by opening them
-  in \texttt{By\ Driver} mode, resulting in being unable to install File System protocols.
+  in \texttt{By\ Driver} mode, resulting in an inability to install File System protocols.
 
-  \emph{Note}: The quirk is mostly relevant for select HP laptops with no drives listed.
+  \emph{Note}: The quirk is mostly relevant for some HP laptops without listed drives.
```

Comment:
This is a bit misworded even originally. Maybe smth similar to: It is relevant for some HP laptops with unsuccessful drive detection resulting in no boot entries.

Reply:
How about...
`This quirk is typically useful on some HP laptops where unsuccessful drive detection results in an absence of boot entries.`

Reply:
Is it even necessary to mention `HP laptops`?
This seems better: `This quirk is useful in cases where unsuccessful drive detection results in an absence of boot entries.`